menu/server_configuration.sh

```diff
@@ -17,41 +17,39 @@ debug_mode
 # Must be root
 root_check
 
+# Set the correct switch for activate_tls
+if [ -f $SCRIPTS/activate-tls.sh ]
+then
+    ACTIVATE_TLS_SWITCH="ON"
+else
+    ACTIVATE_TLS_SWITCH="OFF"
+fi
+
```

Comment:
Why do we need a switch?

Reply:
Because it was on by default but only really makes sense if tls wasn't already activated.

Because of that check (if the activate-tls file is still there) I also don't want to remove the activate-tls file if tls wasn't successfully activated

Reply:
I mean I could of course also come up with a better check. What do you think?

Reply:
Aah I get it, good idea. 👍🏻